apps/dashboard/src/components/contract-components/contract-deploy-form/sequential-token-id-fieldset.tsx

```diff
@@ -0,0 +1,34 @@
+import { FormFieldSetup } from "@/components/blocks/FormFieldSetup";
+import { FormControl } from "@chakra-ui/react";
+import { SolidityInput } from "contract-ui/components/solidity-inputs";
+import type { UseFormRegisterReturn } from "react-hook-form";
+import { FormErrorMessage, FormHelperText, FormLabel } from "tw-components";
+
+interface SequentialTokenIdFieldsetProps {
+  isInvalid: boolean;
+  register: UseFormRegisterReturn;
+  errorMessage: string | undefined;
+}
+
+export const SequentialTokenIdFieldset: React.FC<
+  SequentialTokenIdFieldsetProps
+> = (props) => {
+  return (
+    <FormFieldSetup label="Sequential Token ID">
+      <FormControl isRequired isInvalid={props.isInvalid}>
+        <FormLabel>Start Token ID</FormLabel>
+        <SolidityInput
+          solidityType="uint256"
+          variant="filled"
+          {...props.register}
+        />
+
+        <FormErrorMessage>{props.errorMessage}</FormErrorMessage>
+
+        <FormHelperText className="!text-sm text-muted-foreground">
```

Comment:
If you are using `FormFieldSetup` -
you can remove `FormLabel`, `FormErrorMessage` and `FormHelperText` and just pass those to props `label` , `errorMessage`, `helperText` etc to avoid adding chakra components